### ISSUE TITLE
RenderError

### ISSUE THREAD

**Issue by quangaming2929** [fluddokt/opsu#228]
**Version:** 0.16.0b
**OS:** Linux (armv7l)
**JRE:** The Android Project 0
**OpenGL version:** OpenGL ES 3.1 (ARM)
BOARD: P70-A
FINGERPRINT: Lenovo/P70-A/P70-A:5.1/LMY47D/P70-A_S139_151214_16G_L_ROW:user/release-keys
HOST: scmbuild
MODEL: Lenovo P70-A
INCREMENTAL: P70-A_S139_151214_16G_L_ROW
RELEASE: 5.1

**Error:** `RenderError`
**Stack trace:**
```
java.lang.IllegalArgumentException: keycode cannot be negative, keycode: -1
	at com.badlogic.gdx.Input$Keys.toString(Input.java:248)
	at fluddokt.opsu.fake.Keyboard.getKeyName(Keyboard.java:13)
	at itdelatrisu.opsu.options.Options$GameOption$21.getValueString(Options.java:546)
	at itdelatrisu.opsu.options.OptionsOverlay.renderGenericOption(OptionsOverlay.java:857)
	at itdelatrisu.opsu.options.OptionsOverlay.renderOption(OptionsOverlay.java:751)
	at itdelatrisu.opsu.options.OptionsOverlay.renderOptions(OptionsOverlay.java:691)
	at itdelatrisu.opsu.options.OptionsOverlay.render(OptionsOverlay.java:510)
	at itdelatrisu.opsu.states.SongMenu.render(SongMenu.java:828)
	at fluddokt.opsu.fake.StateBasedGame.render(StateBasedGame.java:134)
	at fluddokt.opsu.fake.GameOpsu.render(GameOpsu.java:150)
	at com.badlogic.gdx.backends.android.AndroidGraphics.onDrawFrame(AndroidGraphics.java:422)
	at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1583)
	at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1286)
```

**Comment by quangaming2929** [comment]
That wil cause when you play opsu! on Android with a mouse connected

**Comment by fluddokt** [comment]
Should be fixed in 0.16.1a
